core-services/prow/02_config/openshift/console-plugin-template/_prowconfig.yaml

```diff
@@ -0,0 +1,22 @@
+tide:
+  queries:
+  - includedBranches:
+    - main
+    - master
```

Comment:
I think we'll want release-* branches so that customers can get the right template for their OpenShift version. (If you're running 4.12, you'd want release-4.12 instead of main which will have changes for 4.13+)

Reply:
@spadgett do you mean that we should have a branch parity like we do in our other repos ? eg. release-4.10, release-4.11, ...

Reply:
Yes, we'll want that. Then plugins can base their work off the release branch that matches the oldest version they need to support. Newer versions of the SDK will have APIs not available in older releases which you could accidentally use.